components/brave_today/common/BUILD.gn

```diff
@@ -1,10 +1,27 @@
 import("//brave/build/config.gni")
+import("//mojo/public/tools/bindings/mojom.gni")
 
 static_library("common") {
   sources = [
+    "pref_names.cc",
+    "pref_names.h",
     "switches.cc",
     "switches.h",
   ]
 
-  deps = [ "//base" ]
+  deps = [
+    "//base",
+    "//brave/components/l10n/browser",
```

Comment:
`common` should not include `browser` side module as it could be included from `renderer`.

Reply:
of course, thanks for the spot